package.json

```diff
@@ -23,6 +23,7 @@
     "moment": "^2.9.0",
     "mousetrap": "^1.4.6",
     "object-path": "^0.9.2",
+    "q": "^1.4.1",
```

Comment:
We need to introduce the `q` dependency in our shrinkwrap too. Could you rebase this PR against master, do an `rm -rf node_modules && npm install` before doing a `npm run shrinkwrap` and add the resulting changes to the `npm-shrinkwrap.json`? 
cc @aldipower for a doublecheck of what I just wrote